holidays/countries/niger.py

```diff
@@ -0,0 +1,314 @@
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars.gregorian import (
+    JAN,
+    FEB,
+    MAR,
+    APR,
+    MAY,
+    JUN,
+    JUL,
+    AUG,
+    SEP,
+    OCT,
+    NOV,
+    DEC,
+    SUN,
+)
+from holidays.groups import ChristianHolidays, InternationalHolidays, IslamicHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_NEXT_MON
+
+
+class Niger(ObservedHolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolidays):
+    """Niger holidays.
+
+    References:
+        * [Law No. 59-22 of December 24, 1959](https://web.archive.org/web/20241106023958/https://www.impots.gouv.ne/media/loi/1960.pdf)
+        * <https://web.archive.org/web/20110721063839/http://www.ais-asecna.org/pdf/gen/gen-2-1/12gen2-1-01.pdf>
+        * <https://web.archive.org/web/20250531032502/https://wageindicator.org/documents/decentworkcheck/africa/niger-french-2021.pdf>
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Niger>
+        * <https://web.archive.org/web/20250531032604/https://wageindicator.org/fr-ne/droit-du-travail/les-conges-et-les-jours-de-repos>
+        * <https://web.archive.org/web/20250206194155/https://www.rivermate.com/guides/niger/leave>
+        * [Eid al-Adha](https://web.archive.org/web/20250117013558/https://www.timeanddate.com/holidays/niger/eid-al-adha)
+        * [Eid al-Fitr](https://web.archive.org/web/20250114130118/https://www.timeanddate.com/holidays/niger/eid-al-fitr)
+        * [Laylat al-Qadr](https://web.archive.org/web/20250531033137/https://www.timeanddate.com/holidays/niger/laylat-al-qadr)
+        * [Islamic New Year](https://web.archive.org/web/20240723135601/https://www.timeanddate.com/holidays/niger/muharram-new-year)
+        * [Prophet's Birthday](https://web.archive.org/web/20250124122731/https://www.timeanddate.com/holidays/niger/prophet-birthday)
+        * [Article 114](https://web.archive.org/web/20240205184429/https://droit-afrique.com/upload/doc/niger/Niger-Code-2012-du-travail.pdf)
+
+    Notes:
+        After Law No. 97-020 of June 20, 1997 establishing public holidays came into
+        effect, holidays that fell on the mandatory weekly rest day (Sunday) were
+        observed on the next Monday.
+    """
+
+    country = "NE"
+    default_language = "ha"
+    supported_languages = ("en_US", "ha")
+    # %s (observed).
+    observed_label = tr("%s (lura)")
+    # %s (estimated).
+    estimated_label = tr("%s (kimanta)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (lura, kimanta)")
+    # Niger gain semi-independence from France on Dec 18 1958.
+    start_year = 1959
```

Comment:
```suggestion
    # Law No. 59-22.
    start_year = 1960
```
According to this Law, from the start holidays were also Eid al-Fitr, Eid al-Adha, Prophet's Birthday, Easter Sunday, Whit Monday.

Reply:
My bad, I don't know how I missed that 😅 
Will fix it soon